### DISCUSSION TITLE
为什么“暂缓处理”不叫“正在投票”

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [Meloong-Git/PCL#2975] [提问] [answered]
“暂缓处理”如果不看说明的话，其实容易和“推迟”搞混（
而且好像所有“暂缓处理”的issue都正在投票中，而且一旦添加这个标签，六角龙机器人就会自动发起投票讨论并且锁定评论区.jpg
“暂缓处理”我感觉表达的意思不太明确

**Comment by burningtnt** [comment]
有点道理，不如直接改成“功能投票”

**Reply by Deep-Dark-Forest** [reply]
还是“正在投票”或者“投票中”比较好，毕竟和“处理中”比较一致

**Comment by MoYuan-CN** [comment]
感觉.. 还行吧..
<暂缓处理> 的 Tooltip 是 <近期不会处理，请在 Discussions 中投票>
<推迟> 的 Tooltip 是 <当前版本暂不处理，之后再说>
看 Tooltip 的话不太容易混

**Reply by MoYuan-CN** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/05c4b1ce-f93e-4425-9871-2b84996f777e)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/d15591c9-1b9d-4ddc-9534-d294e9cde2ad)

**Reply by Deep-Dark-Forest** [reply]
那不还得把鼠标放在上面才能区分开吗（

**Comment by Deep-Dark-Forest** [comment]
@LTCatt

**Comment by WForst-Breeze** [comment]
个人感觉真没必要扣这些细节……
非要硬扣的话，暂缓处理的条目是指在短期内不会制作的内容，扔进投票也只是作为一个开发这个功能优先级的参考之一而已。并不一定是票越高越优先，因此它想强调的是 “暂缓” 而不是 “投票”。

**Reply by Deep-Dark-Forest** [reply]
可是这样的话会给很多刚来pcl2社区的人带来很多误解啊……
难道，“推迟”不也是在短时间以内不会制作的内容吗

**Comment by WorldHim** [comment]
**暂缓** 的释义是 _暂且推迟_，所以从字面意思上来说和 **推迟** 确实差不多。
但是改为 **正在投票** 同样有歧义，可以理解为 **投票** 决定 **是否处理**，而这里的投票已经确认 **会处理**，仅为决定 **处理优先级**，不代表票数低的就 **否决** 该提案

**Comment by LTCatt** [comment, marked as answer]
寻思了一下确实可以改，开了个 issue
